.prettierrc

```diff
@@ -3,7 +3,9 @@
   "trailingComma": "all",
   "arrowParens": "avoid",
   "tabWidth": 2,
+  "useTabs": false,
   "semi": true,
```

Comment:
I wonder if a pre-commit hook might be useful to apply linting and formatting rules?